### PR TITLE
adding live code-coverage for just python39

### PR DESCRIPTION
**Pull request by seankane-msft**
Breaking PR #15577 into two PRs, the latter will focus on the report generation. This PR is just for enabling code coverage

### CODE REVIEW

**Review by scbedd — DISMISSED** [review]

**Comment by scbedd** [comment]
@seankane-msft Can you do me a favor and do a quick search for any other `tests.yml` that invoke a custom matrix of platforms? Extend those with `coverageArg` as well.

**Comment by seankane-msft** [comment]
Addresses #15132

**Review by lmazuel — APPROVED** [review]
Approving the intent, need a engsys approval before merging. Thanks!

File: sdk/schemaregistry/tests.yml [hunk 1]
```diff
@@ -20,12 +20,16 @@ jobs:
...
         Windows_Python27:
           OSVmImage: 'windows-2019'
           PythonVersion: '2.7'
+          CoverageArg: '--disablecov'
         Linux_Python38:
           OSVmImage: 'ubuntu-18.04'
-          PythonVersion: '3.8'
\ No newline at end of file
+          PythonVersion: '3.8'
+          CoverageArg: '--disablecov'
```
Comment (scbedd) [inline sdk/schemaregistry/tests.yml:35]: We want one to remain right?

File: sdk/monitor/tests.yml [hunk 2]
```diff
@@ -16,9 +16,12 @@ jobs:
...
         MacOs_Python37:
           OSVmImage: 'macOS-10.15'
           PythonVersion: '3.7'
+          CoverageArg: '--disablecov'
         Linux_Python39:
           OSVmImage: 'ubuntu-18.04'
-          PythonVersion: '3.9.0'
\ No newline at end of file
+          PythonVersion: '3.9.0'
+          CoverageArg: '--disablecov'
```
Comment (scbedd) [inline sdk/monitor/tests.yml:27]:
```suggestion
          CoverageArg: ''
```

File: sdk/formrecognizer/tests.yml [hunk 3]
```diff
@@ -14,29 +14,37 @@ jobs:
...
             PythonVersion: '3.5'
+            CoverageArg: '--disablecov'
           Windows_Python27:
             OSVmImage: 'windows-2019'
             PythonVersion: '2.7'
+            CoverageArg: '--disablecov'
           Linux_Python39:
             OSVmImage: 'ubuntu-18.04'
             PythonVersion: '3.9.0'
+            CoverageArg: '--disablecov'
```
Comment (scbedd) [inline sdk/formrecognizer/tests.yml:47]:
```suggestion
            CoverageArg: ''
```

File: sdk/eventhub/tests.yml [hunk 4]
```diff
@@ -9,15 +9,19 @@ jobs:
...
           PythonVersion: '3.9.0'
+          CoverageArg: ''
         Windows_Python27:
           OSVmImage: 'windows-2019'
           PythonVersion: '2.7'
+          CoverageArg: '--disablecov'
         MacOs_Python37:
           OSVmImage: 'macOS-10.15'
           PythonVersion: '3.7'
+          CoverageArg: '--disablecov'
```
Comment (scbedd) [inline sdk/eventhub/tests.yml:24]:
```suggestion
          CoverageArg: ''
```

Comment (scbedd) [reply sdk/eventhub/tests.yml:24]: We can chase down owners, but in the meantime, let's get it publishing coverage in general.

File: sdk/schemaregistry/tests.yml (hunk 1 above)
Comment (scbedd) [inline sdk/schemaregistry/tests.yml:35]:
```suggestion
          CoverageArg: ''
```

**Review by scbedd — APPROVED** [review]
Please accept suggestions, otherwise g2g.

**Pull request merged**